### PR TITLE
[FIX] theme_kiddo: refine the theme

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Since the redesign of some snippets, some customizations have been broken. This commit adapts these customizations.
This commit also adjusts some spacing and font sizes.

task-4178078
Part of task-4177978

### CODE REVIEW

File: theme_kiddo/views/snippets/s_banner.xml [hunk 1]
```diff
@@ -4,9 +4,9 @@
 <template id="s_banner" inherit_id="website.s_banner">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="s_parallax_no_overflow_hidden o_full_screen_height oe_img_bg o_bg_img_center pb216 pt72 o_cc o_cc5" remove="pt96 pb96" separator=" "/>
-        <attribute name="data-scroll-background-ratio">0</attribute>
+        <attribute name="class" add="o_full_screen_height oe_img_bg o_bg_img_center pb216 pt72 o_cc o_cc5" remove="pt96 pb96" separator=" "/>
         <attribute name="style">background-image: url("/web/image/website.s_banner_default_image");</attribute>
+        <attribute name="data-oe-shape-data">{'shape':'web_editor/Wavy/22','flip':['x'],'showOnMobile':true,'shapeAnimationSpeed':'0'}</attribute>
```
Comment (agau-odoo) [inline theme_kiddo/views/snippets/s_banner.xml:9]: There is a mismatch with the link, this one doesn't have the c3 color.

File: theme_kiddo/views/snippets/s_three_columns.xml [hunk 2]
```diff
@@ -102,17 +113,21 @@
...
+    <xpath expr="(//figure)[3]" position="attributes">
+        <attribute name="class" add="ratio-1x1" remove="ratio-16x9" separator=" "/>
+    </xpath>
 </template>
 
 <template id="configurator_s_three_columns" inherit_id="website.configurator_s_three_columns">
     <!-- Shape option -->
     <xpath expr="//section" position="attributes">
-        <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/14_001","flip":["y"]}</attribute>
+        <attribute name="data-oe-shape-data">{'shape':'web_editor/Origins/14_001','flip':['y'],'showOnMobile':true,'shapeAnimationSpeed':'0'}</attribute>
```
Comment (agau-odoo) [inline theme_kiddo/views/snippets/s_three_columns.xml:125]: Same thing here, colors don't match the link

**Review by agau-odoo — CHANGES_REQUESTED** [review]
Hey :wave: Thanks for the PR :sunglasses: 

I've left some comments, but otherwise this looks very nice!

File: theme_kiddo/static/description/kiddo_screenshot.jpg
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply theme_kiddo/static/description/kiddo_screenshot.jpg:1]: Good catch ! 🙏 I updated the screenshot 👍

**Comment by Brieuc-brd** [comment]
Thanks for your review @agau-odoo 👍 
I updated the branch, could you have a look ?
The diff : https://github.com/odoo/design-themes/compare/8bb29236030d7087ed45a4421e4dd588cad15aca..d7f4e20a16bd9b0e4af893dca90ccff5e7f845bb

Thanks again !

**Review by agau-odoo — APPROVED** [review]
LGTM :)

@robodoo r+

**Pull request closed without merging**